### PR TITLE
[weather.gismeteo] 0.2.3

### PR DESCRIPTION
**Pull request by vlmaksime**
### Description
Gismeteo Weather Forecast. Real time weather and detailed forecast all round the world
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: weather.gismeteo/default.py [hunk 1]
```diff
@@ -0,0 +1,769 @@
...
+DATEFORMAT     = weather.date_format
+TIMEFORMAT     = weather.time_format
+
+WEATHER_ICON   = weather.weather_icon
+
+KODILANGUAGE   = xbmc.getLanguage().lower()
+
+CURRENT_TIME = {'unix': time.time()}
+
+CACHE_DIR = os.path.join(xbmc.translatePath('special://temp'), weather.id)
```
Comment (romanvm) [inline weather.gismeteo/default.py]: Addons are allowed to store data only in their profile directories. A profile dir can be obtained via `addon.getAddonInfo('profile')` or `addon.config_dir` in SimplePlugin.

**Comment by vlmaksime** [comment]
All comments was fixed

**Comment by romanvm** [comment]
LGTM

File: weather.gismeteo/resources/language/English/strings.po [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+msgid "Location setup"
+msgstr ""
+
+msgctxt "#32102"
+msgid "Advanced"
+msgstr ""
+
+msgctxt "#32103"
+msgid "Expert"
+msgst
```
Comment (ronie) [inline weather.gismeteo/resources/language/English/strings.po]: missing quotes

**Comment by ronie** [comment]
crash due to missing 'mph' string in the language file:
https://paste.ubuntu.com/26353994/

**Comment by vlmaksime** [comment]
@ronie fixed

**Comment by ronie** [comment]
thx!

**Pull request merged**